File: uri/produto/produto.java
import java.io.IOException;
import java.util.*;


public class Main{
    
    public static int produto(int a, int b)
    {
        return ( a * b );
    }

    public static void main( String[] args ) throws IOException
    {
        Scanner input = new Scanner(System.in);
        int a, b, prod;
        a = input.nextInt();
        b = input.nextInt();
        prod = produto(a, b);
        System.out.printf("PROD = %d\n", prod);
    }
}

File: uri/produto/produto.kt
import java.util.Scanner
import kotlin.text.*

fun produto( a : Int, b : Int) : Int {
    return a * b
}

fun main( args: Array< String >){
    var input = Scanner(System.`in`)
    var a : Int = input.nextInt()
    var b : Int = input.nextInt()
    var prod  = produto(a , b)
    println ("PROD = $prod")
}

 